addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/MilightThingState.java

```diff
@@ -11,6 +11,7 @@
 /**
  * Contains a led bulb state including the HSB value, white color temperature and animation values.
  *
+ * @author David Graeff <david.graeff@web.de> - Initial contribution
```

Comment:
you could remove his e-mail here as we do not do this (even if he added it on other classes)